src/main/java/core/basesyntax/service/servise/impl/ReaderImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.service.servise.impl;
+
+import core.basesyntax.service.servise.Reader;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class ReaderImpl implements Reader {
+    private static final String HEADER = "type,fruit,quantity";

```

Comment:
we should be able to reuse Reader service for reading data from any file, now it is not possible
so let's skip the first line in Main or in Parser